### PR TITLE
III-5912 Remove LegacyTitle from codebase

### PR DESCRIPTION
**Pull request by grubolsch**
### Removed
 I removed the LegacyTitle class, and with it some deprecated class that where  no longer used anywhere else.

---
Ticket: https://jira.uitdatabank.be/browse/III-5912

### CODE REVIEW

**Comment by grubolsch** [comment]
Maybe some good info to know is I ran both Place and Event feature tests and everything is still working.

**Comment by LucWollants** [comment]
> Maybe some good info to know is I ran both Place and Event feature tests and everything is still working.

Also `Organizer` code was modified, so I guess that also needs a run

File: src/Model/Import/Offer/Udb3ModelToLegacyOfferAdapter.php [hunk 1]
```diff
@@ -47,10 +47,8 @@ public function getTitle(): Title
     {
         $translatedTitle = $this->offer->getTitle();
 
-        return Title::fromUdb3ModelTitle(
-            $translatedTitle->getTranslation(
-                $translatedTitle->getOriginalLanguage()
-            )
+        return $translatedTitle->getTranslation(
```
[the remark this answers is not in the document]
Comment (grubolsch) [reply src/Model/Import/Offer/Udb3ModelToLegacyOfferAdapter.php]: I remove the entire method away.
I think at some point the entire Udb3ModelToLegacyOfferAdapter.php should go, it seems out dated.
But that is probably out of scope for this PR.

**Comment by grubolsch** [comment]
Organizer also ran - everything still works!

File: src/Http/Event/ImportEventRequestHandler.php [hunk 2]
```diff
@@ -132,7 +132,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         $eventAdapter = new Udb3ModelToLegacyEventAdapter($event);
 
         $mainLanguage = $eventAdapter->getMainLanguage();
-        $title = new Title($eventAdapter->getTitle()->toString());
+        $title = $event->getTitle()->getTranslation($event->getTitle()->getOriginalLanguage());
```
Comment (LucWollants) [inline src/Http/Event/ImportEventRequestHandler.php:135]: For a follow-up pull request:

I think we can improve the `TranslatedValueObject` and give it an extra helper method `getOriginalValue`

**Review by LucWollants — APPROVED** [review]

**Pull request merged**